### DISCUSSION TITLE
cf015749-7450-478b-acb3-3dfdcf140bb5

### DISCUSSION THREAD

**Discussion by giscus** [varofla/blog_giscus#12] [General]
# cf015749-7450-478b-acb3-3dfdcf140bb5

노가다 싫어서 넘어와놓고 제타부터 만들기

https://www.varofla.com/cf015749-7450-478b-acb3-3dfdcf140bb5

**Comment by kimyul02** [comment]
맥북에서 파판 글섭말고 한국서버가 설치가 되나요?

**Reply by varofla** [reply]
![사진](https://www.notion.so/image/https%3A%2F%2Fprod-files-secure.s3.us-west-2.amazonaws.com%2F6ab14156-98e0-4fc6-be53-da5aaa7a261b%2F9153f8a6-af49-452f-96dd-13149a33a591%2F%25E1%2584%2589%25E1%2585%25B3%25E1%2584%258F%25E1%2585%25B3%25E1%2584%2585%25E1%2585%25B5%25E1%2586%25AB%25E1%2584%2589%25E1%2585%25A3%25E1%2586%25BA_2023-11-11_%25E1%2584%258B%25E1%2585%25A9%25E1%2584%258C%25E1%2585%25A5%25E1%2586%25AB_10.07.00.png?id=f34cbafe-53ea-4bfd-891e-d31fd2308c4e&table=block&spaceId=6ab14156-98e0-4fc6-be53-da5aaa7a261b&width=600&userId=18b991dd-bb5e-48f7-9938-74bf431967b0&cache=v2)
패러렐즈로 가상화한 윈도우(ARM)에서 실행하고 있습니다.
M2 울트라(GPU 30코어) 기준 풀옵션은 힘들고 대충 중간정도 하면 숙제용으로 던전 돌 정도가 됩니다 ㅎㅎㅎㅎ
